messages/dji_msgs/msg/Links.msg

```diff
@@ -11,4 +11,12 @@ string HOME_POINT = 'home_point'
 string BASE_LINK = 'base_link'
 string BASE_FLAT = 'base_flat_link'
 string BASE_ENU = 'base_ENU_link'
+string GIMBAL_CAMERA_LINK = 'gimbal_camera_link'
+
+
+# Used by hook_estimator
+string CAMERA_LINK = 'camera_link'
+string WINCH_LINK = 'winch_link'
```

Comment:
This must be published by something. Like the captain or a static transform publisher. So far I've been adding static transforms to the captain for convenience. Can you do that? Follow the example of the gimbal_camera. You just need to change the position from all 0s. In general, whatever captain doesn't produce, is not going to exist in real tests.